content/SCALE/GettingStarted/Migrate/MigratePrep.md

```diff
@@ -8,76 +8,115 @@ tags:
 
+   Migrate [GELI-encrypted pools](https://www.truenas.com/docs/core/coretutorials/storage/pools/storageencryption/#geli-pool-migrations) to a non-GELI-encrypted pool before upgrading from CORE 12.0x or earlier releases!
+
 2. Verify the root user is not locked.
    Go to **Accounts > Users**, use **Edit** for the root user to view current settings and confirm **Lock User** is not selected.
 
-3. After updating to the latest publicly-available release of CORE, download your [system configuration file]({{< relref "UsingConfigurationBackups.md" >}}) and a [debug file]({{< relref "/CORE/UIReference/System/Advanced.md" >}}).
-   Keep these files in a safe place in case you need to revert to CORE with a clean install of the CORE <kbd>iso</kbd> file.
-
-4. Identify your system dataset.
-   If you want to use the same dataset for the system dataset in SCALE, note the pool and system dataset.
+3. Write down or take screenshots of settings to use in the event of a post-upgrade/migration issue or to duplicate in SCALE.
+   Use the checklist below to guide you through this step: 
+
+   <input type="checkbox"> System dataset - Identify your system dataset. If you want to use the same dataset for the system dataset in SCALE, note the pool and system dataset.
+   
+   <input type="checkbox"> Deprecated services - Record the settings for [services deprecated in SCALE](#migrating-from-deprecated-services).
+   
+   <input type="checkbox"> VMs - If you have virtual machines configured in CORE, write down or screenshot network and other setting information.
+   
+   <input type="checkbox"> Plugins or jails - If you use plugins and jails, they do not migrate. Record settings for each plugin/jail and back up the data associated with each.
+   
+   <input type="checkbox"> CAs, certificates, CSRs - If you added certificate authorities, certificates or certificate signing requests to CORE, these should migrate with the system config file, but as a precaution against possible malformed certificates copy or screenshot all CA, certificate, and CSR settings, and save the public and private keys of each. Make sure you have backed-up copies of certificates used in CORE to import or configure in SCALE.
+   
+   <input type="checkbox"> Usernames beginning with (0-9) - Review local user account names and rename or replace these with a letter or underscore before migrating.
    When you set up the first required pool on SCALE import this pool first.
 
-5. Back up your stored data files.
-   If you need to do a clean install with the SCALE <kbd>iso</kbd> file, you can import your data pools into SCALE.
+   <input type="checkbox"> Tunables on CORE - SCALE does not use **Tunables** the way CORE does. SCALE provides script configuration on the **System Settings > Advanced** screen as **Sysctl** scripts. A future release of SCALE could introduce similar tunables options found in CORE but for now it is not available.
```

Comment:
Don't think we want to promise future functionality that hasn't necessarily been scheduled or confirmed yet.
```suggestion
   <input type="checkbox"> Tunables on CORE - SCALE does not use **Tunables** the way CORE does. SCALE provides script configuration on the **System Settings > Advanced** screen as **Sysctl** scripts.
```

Reply:
This was in the article already. I'll reword this.